plugin/trino-hive/src/test/java/io/trino/plugin/hive/optimizer/TestConnectorPushdownRulesWithHive.java

```diff
@@ -102,7 +102,11 @@
     @Override
     protected Optional<LocalQueryRunner> createLocalQueryRunner()
     {
-        baseDir = Files.createTempDir();
+        try {
+            baseDir = Files.createTempDirectory("tmp").toFile();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Please don't swallow exception. Either pass it through or wrap wrap with `UncheckedIOException`. Here and elsewhere.